oclint-core/include/oclint/RuleBase.h

```diff
@@ -26,6 +27,25 @@ class RuleBase
     virtual const std::string attributeName() const {
         return name();
     }
+    virtual const std::string identifier() const {
```

Comment:
Could you move this one to the new line? Also the one for the `attributeName()`?